src/main/java/com/minecolonies/coremod/inventory/InventoryField.java

```diff
@@ -1,5 +1,7 @@
 package com.minecolonies.coremod.inventory;
 
+import com.minecolonies.coremod.util.ItemStackUtils;
+import net.minecraft.entity.player.EntityPlayer;
 import net.minecraft.entity.player.EntityPlayer;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this duplicated import. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)